core/thread.c

```diff
@@ -124,6 +124,19 @@ void thread_add_to_list(list_node_t *list, thread_t *thread)
     list->next = new_node;
 }
 
+void thread_remove_from_list(list_node_t *list, thread_t *thread)
```

Comment:
Agree! Will do :-) now we'll have a PR which depends in this PR which depends in another PR, yeeey :tada: